src/kubernetes_cluster/spec/pod_event/types.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: MIT
+#![allow(unused_imports)]
+use crate::kubernetes_api_objects::spec::{pod::*, resource::*};
+use crate::kubernetes_cluster::spec::message::*;
+use crate::state_machine::action::*;
+use crate::state_machine::state_machine::*;
+use crate::temporal_logic::defs::*;
+use vstd::{multiset::*, prelude::*};
+
+verus! {
+
+pub struct PodEventState {}
```

Comment:
@codyjrivera you can just define `PodEventState` as `()`.

Reply:
This is consistent with how `ClientState` is defined. I will change both.